src/ui/web/link/Link.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Ce fichier est copier de @react-navigation/native et rajoute un hack nécessaire pour react-native-web,
+ * En effet, nous n'avons pas trouvé comment annuler l'événement qui reload la page au click sur l'anchor tag en ReactJS,
```

Comment:
je suis surpris de ce hack

est-ce que tu pourrais ajouter le lien de l'issue https://github.com/necolas/react-native-web/issues/2206 stp ?

Reply:
Pareil, OK pour l'ajout des 2 issues, mais ce n'est pas lié à react-native-web, plutôt à react-navigation.

Reply:
Annnnnnnd it's closed : https://github.com/necolas/react-native-web/issues/2206 :)